connect_ext_test.go

```diff
@@ -1540,9 +1494,7 @@ func TestStreamForServer(t *testing.T) {
 	newPingServer := func(pingServer pingv1connect.PingServiceHandler) (pingv1connect.PingServiceClient, *httptest.Server) {
```

Comment:
Looks like many call sites have been updated to ignore the returned server. Is that maybe _all_ of the call sites? If so, let's remove that return value so this function only returns `PingServiceClient`).